operator.yaml

```diff
@@ -16,11 +16,13 @@ spec:
       containers:
         - name: redis-enterprise-operator
           image: redislabs/operator:498_f987b08
+          securityContext:
```

Comment:
@RoeyPrat we don't need to explicitly set this, we can let k8s determine the exact uid.